content/en/data_jobs/databricks.md

```diff
@@ -44,11 +44,21 @@ Follow these steps to enable Data Jobs Monitoring for Databricks.
 1. On the **Configure** tab, click **Add Databricks Workspace**.
 1. Enter a workspace name, your Databricks workspace URL, account ID, and the client ID and secret you generated.
    {{< img src="data_jobs/databricks/configure-workspace-form-m2m.png" alt="In the Datadog-Databricks integration tile, a Databricks workspace is displayed. This workspace has a name, URL, account ID, client ID, and client secret." style="width:100%;" >}}
+1. For getting visibility on your Databricks costs in Data Jobs Monitoring, provide a System Tables SQL Warehouse ID.
+   - The warehouse must have access to system tables (`system.billing.usage` for example).
+   - Grant your Service Principal the required USAGE and SELECT permissions on:
+      - The SQL Warehouse
+      - The system catalog, schema, and tables
+   - The warehouse should be a Pro or Serverless SQL Warehouse. Classic warehouses are not supported. We recommend setting a 2XS size and configuring Auto Stop after 5–10 minutes.
+
+   See the Datadog [Cloud Cost Management][18] page for full details on the required permissions and setup steps.
```

Comment:
Let's not force the user to go to the CCM docs to get these permissions, we should just list them here (we'll also need more permissions than CCM for lineage and probably other tables). It's ok for it to be in both places.